requirements.txt

```diff
@@ -1,4 +1,4 @@
-amo2kinto==3.1.0
+amo2kinto==3.2.0
```

Comment:
nitpick `amo2kinto>=3.2.0` should be in `setup.py`

Reply:
I added this. I assume I should still/also keep the requirements.txt change this comment references. Let me know if there's anything else!